app/controllers/search_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SearchController < ApplicationController
+  def index
+    location = params[:location]
+    @station = SearchFacade.closest_station(location)
+    @route = RouteFacade.directions(@station.address, location)
```

Comment:
One refactor you could consider is to create a single facade, which would call the RouteService as well as the SearchService and aggregate the data into a PORO. That way, this controller could be simplified and with the relevant data for the user story abstracted into one object there would be fewer poros/facades to test.
I don't have a really strong preference, though, and the advantage of this approach is that these facades might be easier to keep small and encapsulated as the app scales.